partials/schema-prop.html

```diff
@@ -107,7 +107,9 @@
       <div class="children {% if odd %}bg-grey-lightest{% else %}bg-grey-lighter{% endif %} py-4 rounded">
         {% if prop.properties() %}
           {% for pName, p in prop.properties() %}
-            {{ schemaProp(p, pName, odd=(not odd), required=(prop.required() | includes(pName))) }}
+            {% if p.isCircular() === false %}
```

Comment:
What does it render in case it's circular?

Reply:
so far nothing, first we need sampler to support circular refs

Reply:
But sampler is for examples, right? Would be great to have an indicator saying "Circular" or something. And maybe a link to the part of the page where the circular schema is defined.

Reply:
you are right, sorry, was answering without thinking

have a look how it looks after last commit

<img width="1912" alt="Screenshot 2020-08-12 at 12 26 51" src="https://user-images.githubusercontent.com/6995927/90005462-8c492e00-dc97-11ea-893f-f4cf83fa0b73.png">


testing on 
```
asyncapi: '2.0.0'
info:
  title: Example
  version: 0.1.1
channels:
  recursive:
    subscribe:
      message:
        payload:
          $ref: '#/components/schemas/Recursive'
components:
  schemas:
    Recursive:
      type: object
      properties:
        children:
          type: array
          items:
            $ref: '#/components/schemas/Recursive'
        something:
          type: object
          properties:
            maryna:
              $ref: '#/components/schemas/RecursiveNext'
    RecursiveNext:
      type: object
      properties:
        children:
          type: object
          properties:
            maniek:
              $ref: '#/components/schemas/RecursiveNext'
            ulala:
              type: string
```

Reply:
One super UX feature would be to make `[Circular]` a link that points to where the initial `children` is defined. In this case, it's clear but in large objects might not be that easy to find.

Reply:
this is the latest view
<img width="1621" alt="html" src="https://user-images.githubusercontent.com/6995927/90247151-4eced700-de36-11ea-8c50-5e62f7497342.png">

by link you mean like hyperlink? in this example where would it point?